### PR TITLE
Fix cuda version detect on Windows

### PR DESCRIPTION
**Pull request by meteorcloudy**
On Windows, the name of the shared libraries is like:
cudart64_80.dll and cudnn64_5.dll
@davidzchen @gunan

### CODE REVIEW

File: third_party/gpus/cuda_configure.bzl [hunk 1]
```diff
@@ -194,13 +194,13 @@ def _cuda_version(repository_ctx, cuda_toolkit_path, cpu_value):
     auto_configure_fail(
         "CUDA version detected from nvcc (%s) does not match " +
         "TF_CUDA_VERSION (%s)" % (version, environ_version))
-  return version
+  return ("64_" + version.replace(".", "")) if cpu_value == "Windows" else version
```
Comment (gunan) [inline third_party/gpus/cuda_configure.bzl]: To increase readability, could you replace ternary operator with an if else block?

File: third_party/gpus/cuda_configure.bzl [hunk 2]
```diff
@@ -236,9 +236,9 @@ def _cudnn_version(repository_ctx, cudnn_install_basedir):
     environ_version = repository_ctx.os.environ[_TF_CUDNN_VERSION].strip()
   if environ_version and version != environ_version:
     auto_configure_fail(
-        "cuDNN version detected from %s (%s) does not match " +
-        "TF_CUDNN_VERSION (%s)" % (str(cudnn_h_path), version, environ_version))
-  return version
+        ("cuDNN version detected from %s (%s) does not match " +
+        "TF_CUDNN_VERSION (%s)") % (str(cudnn_h_path), version, environ_version))
+  return ("64_" if cpu_value == "Windows" else "") + version
```
Comment (gunan) [inline third_party/gpus/cuda_configure.bzl]: Same here, to increase readability, can we have an if else block?

**Review by gunan — COMMENTED** [review]
Two comments, otherwise looks good.

File: third_party/gpus/cuda_configure.bzl (hunk 1 above)
Comment (meteorcloudy) [inline third_party/gpus/cuda_configure.bzl]: Good point! Done.

**Comment by gunan** [comment]
Jenkins, test this please.

**Review by gunan — APPROVED** [review]

**Comment by meteorcloudy** [comment]
Jenkins, test this please.

**Comment by gunan** [comment]
The multiple runs seem to have covered all flaky tests. I will merge this now.

**Pull request merged**